WooCommerce/WooCommerceTests/ViewRelated/SwiftUI/BoldableTextViewTests.swift

```diff
@@ -0,0 +1,69 @@
+import SwiftUI
+import XCTest
+@testable import WooCommerce
+
+final class BoldableTextViewTests: XCTestCase {
+    func test_content_without_matching_asterisks() throws {
+        // When
+        let view = BoldableTextView("I will *never stop learning.")
+
+        // Then
+        let text = try XCTUnwrap(view.body as? Text)
+        let expectedText = Text("") + Text(verbatim: "I will *never stop learning.")
```

Comment:
This got me stuck for a while: in SwiftUI, `Text(content)` where `let content = "string"` is not equal to `Text("string")`. To compare to a static string, I had to use `Text(verbatim: "string")` 🤯